source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -30,9 +30,10 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::filter>,
 public:
   AuthenticatorImpl(const CheckAudience* check_audience,
                     const absl::optional<std::string>& provider, bool allow_failed,
-                    JwksCache& jwks_cache, Upstream::ClusterManager& cluster_manager)
-      : jwks_cache_(jwks_cache), cm_(cluster_manager), check_audience_(check_audience),
-        provider_(provider), is_allow_failed_(allow_failed) {}
+                    JwksCache& jwks_cache, Upstream::ClusterManager& cluster_manager,
+                    CreateJwksFetcherCb createJwksFetcherCb)
```

Comment:
per style guide: variable name is lower_case

Reply:
This is from the merged code. I'll change it